services/cards-consultation/src/main/java/org/opfab/cards/consultation/repositories/ArchivedCardCustomRepositoryImpl.java

```diff
@@ -308,6 +318,17 @@ private boolean isLatestUpdateOnlyParamPresent(MultiValueMap<String, String> par
         return false;
     }
 
+    private boolean isAdminModeParamPresent(CurrentUserWithPerimeters currentUserWithPerimeters,
```

Comment:
I would name it checkIfInAdminMode as the method is not only checking the param but as well checking the user is admin 
I would put a log with warning level if we want to be in admin mode but we are not member of the admin group

Reply:
Done.